website/blog/2022-07-23-progress-so-far-outreachy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Progress so far - 2022 Outreachy interns
+description: Contributions and progress made by the Outreachy interns (2022)
+tags: ['moja-global','outreachy','open-source','internships','mentorship']
+authors: Namyalg
+slug: progress-so-far-outreachy
+hide_table_of_contents: false
+---
+
+[Kateryna Derhaus](https://github.com/derha) and 
+[Ankita Sharma](https://github.com/ankitaS11) are GSoC mentees for the year 2022. 
```

Comment:
Can you recheck this? @Namyalg

Reply:
Yes, thanks for pointing it out, I missed it